### ISSUE TITLE
getSensors() possibily needs to return multiple metric types

### ISSUE THREAD

**Issue by do3meli** [maglub/piLogger#178]
currently the `getSensors()` function in `myfunctions.inc.php` is only returning information for one "metric". one should be able to retrieve all metric information for a particular sensor address/id.

**Comment by maglub** [comment]
This is correct, and it requires to add one more abstraction, i.e "package" for chips that has one ID but more than one sensor. This is the case with, for example, counters with 2 counters in the same chip, or chips with a termometer and a humidity sensor.
